plasmapy/particles/tests/test_ionization_state.py

```diff
@@ -745,3 +791,9 @@ def test_iteration_with_nested_iterator():
         for fraction2 in hydrogen:
             i += 1
     assert i == 4
+
+
+def test_ionization_state_is_another():
+    hydrogen = IonizationState("D+", n_elem=1e20 * u.m ** -3, T_e=10 * u.eV)
+    deuterium = IonizationState("T+", n_elem=1e20 * u.m ** -3, T_e=10 * u.eV)
+    assert hydrogen is not deuterium
```

Comment:
```suggestion
def test_ionization_state_inequality_and_identity():
    deuterium_states = IonizationState("D+", n_elem=1e20 * u.m ** -3, T_e=10 * u.eV)
    tritium_states = IonizationState("T+", n_elem=1e20 * u.m ** -3, T_e=10 * u.eV)
    assert deuterium_states is not tritium_states
    assert deuterium_states != tritium_states
```
Should at least make the variable names consistent with the isotopes.

Reply:
Gah. Yeah. Duh. Of course.